index.md

```diff
@@ -26,6 +26,6 @@ This default behavior can be adapted or controlled by a larger management system
 
 1. It should always be possible to factory reset back to either the known built behavior of the system or roll back to previous behavior if an updated image does not function correctly.
 
-1. A cryptographic trust chain that runs from the hardware, through the boot loader, through the operating system all the way to the applications ensures that only the expected code is run, and the contents of the operating system and applications have not been changed unexpectedly.
+1. It is supported to create a cryptographic trust chain that runs from the hardware, through the boot loader, through the operating system all the way to the applications ensures that only the expected code is run, and the contents of the operating system and applications have not been changed unexpectedly.
```

Comment:
How about "The system supports" instead of "It is supported" ... the latter seems to imply that one can establish a cryptographic trust chain today. I don't think that's yet the case, it's not fully implemented. WDYT?

Reply:
There's parts that work here by dropping down to raw ostree and signing, but yes.

Reply:
ISTM we're a bit inconsistent here in that some of the goals are indicative ("The system updates atomically." - no "should") vs these last 3 are more subjunctive.  I matched the previous one in using "should".